hack/verify-examples.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+cd "${OS_ROOT}"
+
+echo "===== Verifying Generated Examples Bindata ====="
+
+TMP_GENERATED_EXAMPLES_DIR="_output/verify-examples-bindata"
+
+echo "Generating examples bindata..."
+if ! output=`OUTPUT_ROOT=${TMP_GENERATED_EXAMPLES_DIR} ${OS_ROOT}/hack/build-examples.sh 2>&1`
+then
+	echo "FAILURE: Generation of fresh bindata failed:"
+	echo "$output"
+  exit 1
+fi
+
+echo "Diffing current examples bindata against freshly generated bindata"
+ret=0
+diff -Nap "${OS_ROOT}/pkg/examples/bindata.go" "${TMP_GENERATED_EXAMPLES_DIR}/pkg/examples/bindata.go" &> /dev/null || ret=$?
```

Comment:
I don't like "examples" as the word here.  It's data related to bootstrapping a cluster, or initializing it, or represents system defaults.  Examples doesn't really work there.